mir_eval/util.py

```diff
@@ -715,7 +720,7 @@ def filter_kwargs(function, *args, **kwargs):
     function_args = inspect.getargspec(function).args
     # Construct a dict of those kwargs which appear in the function
     filtered_kwargs = {}
-    for kwarg, value in kwargs.items():
+    for kwarg, value in list(kwargs.items()):
```

Comment:
I'd be in favor of it; seems like everything else uses six anyway (even if scipy uses an internal copy).  But it's @craffel's call.